spec/features/users_show_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+RSpec.describe User, type: :system do
+  describe 'show user page' do
+    it 'I can see the users profile picture.' do
+      visit user_path(1)
+      expect(page).to have_xpath("//img[@src='https://unsplash.com/photos/F_-0BxGuVvo']")
+    end
+
+    it 'I can see the number of posts each user has written.' do
+      visit users_path(1)
+      expect(page).to have_content('Number of posts: 1')
+    end
+
+    it 'I can see the username' do
+      visit users_path(1)
+      expect(page).to have_content('Tom')
+    end
+
+    it 'I can see the users bio.' do
+      visit user_path(1)
+      expect(page).to have_content('Teacher from Mexico.')
+    end
+
+    it 'I can see the users first 3 posts.' do
+      @posts = Post.where(author_id: 1).limit(3).order(created_at: :desc)
+      @posts.each do |_post|
+        visit user_path(1)
+        expect(page).to have_content('Text: This is my first post')
+      end
+    end
+
+    it 'I can see a button that lets me view all of a users posts.' do
+      visit user_path(1)
+      expect(page).to have_content('See all posts')
+    end
+
+    it 'When I click to see all posts, it redirects me to the users posts index page.' do
+      visit user_path(1)
+      click_link 'See all posts'
+      expect(page).to have_current_path(user_posts_path(1))
+    end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=When%20I%20click%20a%20user%27s%20post%2C%20it%20redirects%20me%20to%20that%20post%27s%20show%20page.), Please add a tests that redirects to the post's show page when you click on a user's post.